fastlane/lanes/build.rb

```diff
@@ -0,0 +1,427 @@
+# frozen_string_literal: true
+
+# Lanes related to Building and Testing the code
+#
+platform :ios do
+  # Builds the WordPress app for Testing
+  #
+  # @called_by CI
+  #
+  desc 'Build WordPress for Testing'
+  lane :build_for_testing do |options|
+    run_tests(
+      workspace: WORKSPACE_PATH,
+      scheme: 'WordPress',
+      derived_data_path: DERIVED_DATA_PATH,
+      build_for_testing: true,
+      device: options[:device],
+      deployment_target_version: options[:ios_version]
+    )
+  end
+
+  # Builds the Jetpack app for Testing
+  #
+  # @called_by CI
+  #
+  desc 'Build Jetpack for Testing'
+  lane :build_jetpack_for_testing do |options|
+    run_tests(
+      workspace: WORKSPACE_PATH,
+      scheme: 'Jetpack',
+      derived_data_path: DERIVED_DATA_PATH,
+      build_for_testing: true,
+      deployment_target_version: options[:ios_version]
+    )
+  end
+
+  # Runs tests without building the app.
+  #
+  # Requires a prebuilt xctestrun file and simulator destination where the tests will be run.
+  #
+  # @option [String] name The (partial) name of the `*.xctestrun` file to run
+  # @option [String] device Name of the simulator device to run the test on
+  # @option [String] ios_version The deployment target version to test on
+  #
+  # @called_by CI
+  #
+  desc 'Run tests without building'
+  lane :test_without_building do |options|
+    # Find the referenced .xctestrun file based on its name
+    build_products_path = File.join(DERIVED_DATA_PATH, 'Build', 'Products')
+
+    test_plan_path = Dir.glob(File.join(build_products_path, '*.xctestrun')).select do |e|
+      e.include?(options[:name])
```

Comment:
I'm not sure what `e` stands for? 🤔 

What do you think of this? 

```suggestion
    test_plan_path = Dir.glob(File.join(build_products_path, '*.xctestrun')).select do |path|
      path.include?(options[:name])
```

Reply:
Sounds good to me.
Didn't want to change the implementations too much to not make the review of this PR harder that it already is, and make my script producing the lane diffs make minimalistic diffs for easier review; but since you suggested the change here and thus will be aware about it when reviewing the diff.txt files I guess I can commit the suggestion right away 🙂